docker-compose.yml

```diff
@@ -234,6 +234,7 @@ services:
       - COUNTRY=${COUNTRY:-bgd}
       - HOST=0.0.0.0
       - COUNTRY_CONFIG_URL=http://countryconfig:3040
+      - USER_MANAGEMENT_URL=http://user-mgnt:3030/
```

Comment:
https://github.com/opencrvs/e2e/blob/develop/infrastructure/docker-compose.app.yml
will need updating for e2e's to work